.github/ISSUE_TEMPLATE/02-name-correction.yml

```diff
@@ -8,20 +8,29 @@ body:
   - type: markdown
     attributes:
       value: >
-        This form will report issues with author pages—for example, if a name needs to be disambiguated,
-        or if two versions of a name should be mapped to the same individual.
-        For cases where an individual has published under multiple name variants, submitting a pull request
-        that modifies [`name_variants.yaml`](https://github.com/acl-org/acl-anthology/blob/master/data/yaml/name_variants.yaml)
-        directly will expedite the process. If an author name listed for a paper does not match
-        what is in the PDF, instead use the "Fix data" button on the paper page,
-        which will help automate the correction. Thanks!
+        This form will report issues with author pages. There are two situations: (1) merging and (2) splitting.
+        Merging allows papers published under different names or name variants to be displayed under a single
+        canonical variant of the name, while splitting allows the separation of papers published under the same
+        name but belonging to different people.
```

Comment:
I feel this reads very terminology-dense and may be confusing to people not so intimately familiar with our nomenclature. One suggestion is to say "Merging allows ... to be displayed under a single author page". People understand "author page", they may not understand "canonical variant of a name".

Reply:
Yeah, that's dense, I tried to simplify it